### ISSUE TITLE
Expose AWS postgres instance and default db properties 

### ISSUE THREAD

**Issue by DanielJRutledge** [rmoscowitz/houston-elibrary-portal#6]
Expose the EC2 Postgres instance and have the postgres properties default to that as opposed to a local instance. This way contributors don't have to have a db dump to contribute anything to server-side development.

**Comment by DanielJRutledge** [comment]
Thought: we will probably have to tweak the postgres user to read-only if we publish the authentication credentials.

**Comment by emptyflash** [comment]
I agree with this, especially because we're not doing anything on the frontend or api to change the database as of right now. In the far future it'd probably be best to have a dev and stage environment for that kind of thing and/or make it trivial to set up and populate local dbs.
